src/Core/Environment.php

```diff
@@ -151,4 +166,92 @@ public static function getTimeLimitMax()
     {
         return static::$timeLimitMax;
     }
+
+    /**
+     * Get value of environment variable
+     *
+     * @param string $name
+     * @return mixed Value of the environment variable, or null if not set
+     */
+    public static function getenv($name)
+    {
+        switch (true) {
+            case array_key_exists($name, static::$env):
+                return static::$env[$name];
+            case array_key_exists($name, $_ENV):
+                return $_ENV[$name];
+            case array_key_exists($name, $_SERVER):
+                return $_SERVER[$name];
+            default:
+                return getenv($name);
+        }
+    }
+
+    /**
+     * Set environment variable using php.ini syntax.
+     * Acts as a process-isolated version of putenv()
+     * Note: This will be parsed via parse_ini_string() which handles quoted values
+     *
+     * @param string $string Setting to assign in KEY=VALUE or KEY="VALUE" syntax
+     */
+    public static function putenv($string)
+    {
+        // Parse name-value pairs
+        $envVars = parse_ini_string($string) ?: [];
+        foreach ($envVars as $name => $value) {
+            self::setEnv($name, $value);
+        }
+    }
+
+    /**
+     * Load environment variables from .env file
+     *
+     * @param string $path Path to the file
+     * @return array List of values parsed as an associative array
+     */
+    public static function putenvFromFile($path)
```

Comment:
Why have we reimplemented dotenv?

Reply:
Pretty much!

Reply:
Wouldn't it be better to retain a dotenv dependency (either vlucas or symfony) and make reference to its loader within `Env`?

Reply:
I'll have a look into symfony.

Reply:
Symfony seems to do exactly the same thing; symfony dotenv simply uses core-php `getenv`. Given our implementation is small and works for us, I suggest to leave it as is.

Reply:
ok